linodecli/cli.py

```diff
@@ -462,6 +99,9 @@ def load_baked(self):
                 if "_spec_version" in self.ops:
                     self.spec_version = self.ops["_spec_version"]
                     del self.ops["_spec_version"]
+                if "_spec" in self.ops:
+                    self.spec = self.ops["_spec"]
+                    del self.ops["_spec"]
```

Comment:
```suggestion
                    self.spec = self.ops.pop("_spec")
```
A similar change can also apply to `_base_url` and `_spec_version` above.